packages/engine/src/scene/components/MediaComponent.ts

```diff
@@ -291,6 +295,18 @@ export function MediaReactor() {
     [media.paused, mediaElement]
   )
 
+  useEffect(
+    function updateSeekTime() {
+      console.log('DEBUG Triggred seek')
+      if (!mediaElement || media.seekTime.value < 0 || mediaElement.element.value.currentTime === media.seekTime.value)
+        return
+      mediaElement.element.currentTime.set(media.seekTime.value) // set time and stop playing
+      media.paused.value ? media.paused.set(false) : mediaElement.element.value.play() // start play again
+      media.seekTime.set(-1)
```

Comment:
Why do we set seekTime to -1 at this point?

Reply:
if we keep the time as is, then if we want to reset the next time (by setting to a fixed value of .001) the reactor wont run